test/jobs/daily_broadcast_auctions_job_test.rb

```diff
@@ -0,0 +1,31 @@
+require 'test_helper'
+
+class DailyBroadcastAuctionsJobTest < ActiveJob::TestCase
+  include ActionMailer::TestHelper
+  def setup
```

Comment:
Use callback instead (preferred) or call `super`, otherwise parent method is not called.

Reply:
`super` is already present in setup method, am I missing something? What's the benefit of preferring callback?